nvidia_nims/vista_3d_remote_nim.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        isotropy = (
</pre><p>Here I'd recommend saving the spacing to a variable: <code>input_spacing = nput_image.GetSpacing()</code> and then use that variable everywhere you're calling <code>input_image.GetSpacing()</code>. This reduces the code a bit and helps with readability.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>